

Here are the [KnowEnG](https://knoweng.org/) repos.  For more
information about the KnowEnG platform, or to sign up for an account,
visit [the KnowEnG site](https://knoweng.org/analyze/).

## Platform Interface
 - [platform](https://github.com/KnowEnG/platform): Web services and client for the KnowEnG platform
 - [quickstart-demos](https://github.com/KnowEnG/quickstart-demos): Sample files and readmes for analytics pipelines on KnowEnG platform

## Analysis Pipeline
 - [GeneSet_Characterization_Pipeline](https://github.com/KnowEnG/GeneSet_Characterization_Pipeline): Ranks a user supplied gene set against KnowEnG's gene sets collection
 - [Gene_Prioritization_Pipeline](https://github.com/KnowEnG/Gene_Prioritization_Pipeline): Network-based prioritization of genes associated with phenotypes
 - [Feature_Prioritization_Pipeline](https://github.com/KnowEnG/Feature_Prioritization_Pipeline): General purpose features prioritization pipeline
 - [Simplified_InPheRNo_Pipeline](https://github.com/KnowEnG/Simplified_InPheRNo_Pipeline)
 - [Samples_Clustering_Pipeline](https://github.com/KnowEnG/Samples_Clustering_Pipeline): Clusters the sample-based columns of a spreadsheet whose rows correspond to genes
 - [General_Clustering_Pipeline](https://github.com/KnowEnG/General_Clustering_Pipeline): Clusters columns of any data frame
 - [Clustering_Evaluation](https://github.com/KnowEnG/Clustering_Evaluation): Evaluates the clustering result of KnowEnG's Samples Clustering Pipeline
 - [Signature_Analysis_Pipeline](https://github.com/KnowEnG/Signature_Analysis_Pipeline): Gene signature similarity analysis
 - [Phenotype_Prediction_Pipeline](https://github.com/KnowEnG/Phenotype_Prediction_Pipeline): Regression to infer an 'omic'-phenotype association
 - [Spreadsheets_Transformation](https://github.com/KnowEnG/Spreadsheets_Transformation): Apply various transformations (transpose, merge, ....) on one or more DataFrames
 - [Data_Cleanup_Pipeline](https://github.com/KnowEnG/Data_Cleanup_Pipeline): Maps gene-label row names of a spreadsheet to Ensembl names and checks data formats
 - [KnowEnG_Pipelines_Library](https://github.com/KnowEnG/KnowEnG_Pipelines_Library): Python scripts to support KnowEnG pipelines

## Common Workflow Language (CWL) Descriptions
 - [KnowEnG_CWL](https://github.com/KnowEnG/KnowEnG_CWL): CWL for KnowEnG pipelines for cwltool and CGC

## Knowledge Network
 - [KnowNet_Pipeline_Tools](https://github.com/KnowEnG/KnowNet_Pipeline_Tools): Setup resources and start KN_Builder
 - [KN_Fetcher](https://github.com/KnowEnG/KN_Fetcher): Download a subnetwork from the KnowEnG Knowledge Network
 - [KN_Mapper](https://github.com/KnowEnG/KN_Mapper): Maps names to entity IDs in the KnowEnG Knowledge Network
 - [KN_Builder](https://github.com/KnowEnG/KN_Builder): Fetch and parse source files to construct KnowEnG Knowledge Network

## Education
 - [TeachEnG](https://github.com/KnowEnG/TeachEnG): A Teaching Engine for Genomics
 - [ClusterEnG](https://github.com/KnowEnG/ClusterEnG): Educational web resource for clustering and visualizing high-dimensional data
 - [SequencEnG](https://github.com/KnowEnG/SequencEnG): An interactive learning resource for next-generation sequencing (NGS) techniques

## Research
 - [DRaWR](https://github.com/KnowEnG/DRaWR): Discriminative Random Walk with Restart
 - [ProGENI](https://github.com/KnowEnG/ProGENI): Prioritization of Genes Enhanced with Network Information
 - [pgenmi](https://github.com/KnowEnG/pgenmi): Probabilistic Gene Expression in the Middle Analysis
 - [clusDCA](https://github.com/KnowEnG/clusDCA): Software to accompany ISMB2015 paper
 - [PACER](https://github.com/KnowEnG/PACER): PACER tool for pathway identification from chemosensitivity data

